### PR TITLE
/explore page

### PR DESCRIPTION
**Pull request by Kriesse**
A page to explore all public datasets.

### CODE REVIEW

**Comment by karissa** [comment]
It seems a bit weird to me that I can click on the name of the dat and also the 'preview' button. What if we make the whole thing clickable?

For the example datasets, want to use the same ones we've been using for now?

I know we also want to implement peer count and dat size, but let's do that in a separate PR to push this through since that isn't crucial at the moment.

**Comment by Kriesse** [comment]
@karissa The entire item is now clickable and points to the Dat Preview page, except when clicking on "Copy Link". Also cleaned up css a bit.

image

**Comment by juliangruber** [comment]
the tests that aren't passing are testing against currently deployed `datproject.org`, so they shouldn't have anything to do with this branch and I think it's fine to merge

**Comment by karissa** [comment]
@juliangruber The tests should pass here before merging into master, they are using a docker container for this branch, not against datproject.org

**Comment by juliangruber** [comment]
@karissa oh, gotcha! just pushed another fix

**Comment by karissa** [comment]
Nice looks like the PR branch passed which is enough for me

File: client/js/pages/landing/header.js [hunk 1]
```diff
@@ -3,17 +3,17 @@ var html = require('choo/html')
...
             <a href="/" class="dat-logo">
               <img src="/public/img/dat-hexagon.svg" />
               <span class="dat-logo__word">Dat</span>
             </a>
             <nav class="main-nav">
+              <a href="/explore" class="header-nav-link">Explore</a>
               <a href="/install" class="header-nav-link">Install</a>
               <a href="/about" class="header-nav-link">About</a>
               <a href="http://blog.datproject.org" class="header-nav-link">Blog</a>
-              <a href="/team" class="header-nav-link">Team</a>
```
Comment (karissa) [inline client/js/pages/landing/header.js]: nice choice!

File: client/js/pages/archive/index.js [hunk 2]
```diff
@@ -63,7 +63,7 @@ const archivePage = (state, prev, send) => {
           ${error(state.archive.error)}
           <div class="dat-details">
             <div id="permissions" class="dat-detail">
-              ${permissions({owner: owner})}
+              xx ${permissions({owner: owner})}
```
Comment (karissa) [inline client/js/pages/archive/index.js]: what's this xx?

File: client/js/elements/message/index.js [hunk 3]
```diff
@@ -1,8 +1,28 @@
...
+    }
+    &.warning {
+      background-color: var(--color-yellow-hover);
+    }
+  }
+`
 module.exports = function (props) {
   if (!props.message) return html``
-  return html`<div class="message ${props.type}">
+  return html`<div class="${messageStyles} ${props.type}">
```
Comment (karissa) [inline client/js/elements/message/index.js]: is there a respective scss file we can remove by doing this?

Comment (Kriesse) [inline client/js/elements/message/index.js]: no, this just replaced a class, not a file

File: client/js/pages/archive/index.js (hunk 2 above)
Comment (Kriesse) [inline client/js/pages/archive/index.js]: whoops, leftover from testing. removed it.

File: client/js/pages/auth/profile.js [hunk 4]
```diff
@@ -36,7 +36,7 @@ module.exports = (state, prev, send) => {
       <div class="flex flex-column flex-row-m flex-row-l ${profileStyles}">
         <div class="bg-neutral-04 pa4 tc tl-m tl-l">
           <div class="name">
-            <h1 class="f4 mb1">Kristina Schneider${name}</h1>
```
Comment (karissa) [inline client/js/pages/auth/profile.js]: hehehe

**Comment by millette** [comment]
The new [dat cli (13.9.0) mentions the explore page](https://github.com/datproject/dat/blob/master/src/commands/publish.js#L47) but it doesn't seem to exist on the new registry.

**Pull request merged**